### PR TITLE
add me maintainer / Fix Python version 

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
Found this on the original repo : https://github.com/LuminosoInsight/python-ftfy/issues/131

I will fix the python version to 3.8, look like python 3.9.1 updated unicode again. @chrisburr 
[`BPO 39926, Unicode support updated to version 13.0.0;`](https://www.python.org/downloads/release/python-391/)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,10 +19,10 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python ==3.8.1
```
Comment (chrisburr) [inline recipe/meta.yaml:22]:
```suggestion
    - python >=3.6,<3.9
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,10 +19,10 @@ build:
...
     - pip
   run:
-    - python >=3.6
+    - python ==3.8.1
```
Comment (chrisburr) [inline recipe/meta.yaml]:
```suggestion
    - python >=3.6,<3.9
```

**Comment by FernandezMathieu** [comment]
We're good @chrisburr , can you merge ?

**Comment by chrisburr** [comment]
Thanks @FernandezMathieu!

**Pull request merged**